components/script/dom/eventsource.rs

```diff
@@ -105,17 +105,19 @@ impl EventSourceContext {
         }
         let global = event_source.global();
         let event_source = self.event_source.clone();
-        // FIXME(nox): Why are errors silenced here?
-        let _ = global.remote_event_task_source().queue(
-            task!(announce_the_event_source_connection: move || {
-                let event_source = event_source.root();
-                if event_source.ready_state.get() != ReadyState::Closed {
-                    event_source.ready_state.set(ReadyState::Open);
-                    event_source.upcast::<EventTarget>().fire_event(atom!("open"));
-                }
-            }),
-            &global,
-        );
+        global
```

Comment:
This is called from the IPC router thread, and I wouldn't assert here unless we add some guarantees that it is shutdown when the other threads in the process shutdown.